nautobot/extras/templates/extras/dynamicgroup_edit.html

```diff
@@ -0,0 +1,26 @@
+{% extends "generic/object_edit.html" %}
+{% load form_helpers %}
+
+{% block form %}
+    <div class="panel panel-default">
+        <div class="panel-heading"><strong>Dynamic Group</strong></div>
+        <div class="panel-body">
+            {% render_field form.name %}
+            {% render_field form.slug %}
+            {% render_field form.description %}
+            {% render_field form.content_type %}
+        </div>
+    </div>
+   {% if filter_form %}
+    <div class="panel panel-default">
+        <div class="panel-heading"><strong>Filter Fields</strong></div>
+        <div class="panel-body">
+            <span class="help-block">
+                Select the filtering criteria to determine membership of objects matching the Content Type for this Dynamic Group. Fields that are not a dropdown are expected to have string inputs and do not support multiple values.
```

Comment:
The lack of multi-value support on text fields seems like a pretty significant limitation to me. So long as we have it well documented for this initial release then OK.

Reply:
I've not found an existing use-case for that. All the multi-value fields I've seen are related fields OR are constrained to choices. In the event of say, `DeviceFilterSet.name` being a multiple-value field, how would we constrain the choices? Can you think of one?

I can think of a legit longer term solution that actually has a specialized auto-complete that legit constraints to a list of `Device.name` values. We might be able to do a queryset for that or something? But it seems complicated.